### ISSUE TITLE
Linear to Gaussian

### ISSUE THREAD

**Issue by hydodie** [eatthatpie/covid-gauss#3]
hello I like the program that you made https://covid-gauss.site/, I also want to make something like that

I want to ask if you can help me to make a transformation from linear to gaussian, because I was very difficult to make it and the information that I was looking for did not help me to complete the task,

`from sklearn import datasets, linear_model
from sklearn.metrics import mean_squared_error, r2_score
from sklearn.linear_model import LinearRegression
df1=dataitaly[['time', 'ConfirmedCases']]

from sklearn.utils import shuffle
df1 = shuffle(df1,random_state=0)

X=np.array(df1[['time']])
y=np.array(df1[['ConfirmedCases']])

X_train = X[:-10]
X_test = X[-10:]

y_train = y[:-10]
y_test = y[-10:]

regr = linear_model.LinearRegression()

regr.fit(X_train, y_train)

y_pred = regr.predict(X_test)

print('Coefficients: \n', regr.coef_)

print('Mean squared error: %.2f'
      % mean_squared_error(y_test, y_pred))

print('Coefficient of determination: %.2f'
      % r2_score(y_test, y_pred))

plt.scatter(X_test, y_test,  color='black')
plt.scatter(X_train, y_train,  color='black')
plt.plot(X_test, y_pred, color='blue', linewidth=3)

plt.xticks(())
plt.yticks(())

plt.show()`

regards
